examples/dax/subnetgroup.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: dax.aws.crossplane.io/v1alpha1
+kind: SubnetGroup
+metadata:
+  name: example-subnet-group
+spec:
+  forProvider:
+    region: us-east-1
+    description: "Test subnet group description."
+    subnetIds:
+      - "subnet-0999fa999bef99a99"
+---
```

Comment:
```suggestion
```

Reply:
Thanks, fixed in [`0aac4`](https://github.com/crossplane/provider-aws/pull/1256/commits/0aac42d58a30c4e20b55d6810dc00363ba1ab2c2).